common/tools/dev-tool/src/commands/run/index.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license
+
+import { subCommand, makeCommandInfo } from "../../framework/command";
+
+export const commandInfo = makeCommandInfo("run", "run scripts such as test:node");
```

Comment:
nit: `test:node` not accurate if the sub commands include both node and browser?